### DISCUSSION TITLE
OTA from 4.0 beta3 to beta4

### DISCUSSION THREAD

**Discussion by akennerly** [forkineye/ESPixelStick#437] [Q&A] [answered]
I'm trying to perform an OTA update from beta3 to beta4.  I have the .efu file that was generated with the java tool. After uploading the beta4 .efu, the web UI still shows beta3 and Oct 2, 2021 build date.

Is it possible to perform OTA updates between the beta firmwares?

Hardware is the ESPixelStickV3

**Comment by forkineye** [comment, marked as answer]
EFU updates aren't supported in 4.0 yet.

**Reply by DrAnonymous** [reply]
Should an issue be created that says EFU isn't working yet? Is there a list of features planned for v4.0?

**Reply by forkineye** [reply]
Re-opened the issue here - https://github.com/forkineye/ESPixelStick/issues/446
